adsft/tasks.py

```diff
@@ -20,14 +21,16 @@
     Queue('extract-grobid', app.exchange, routing_key='extract-grobid'),
     Queue('output-results', app.exchange, routing_key='output-results'),
     Queue('facility-ner', app.exchange, routing_key='facility-ner'),
+    Queue('nlp-techniques', app.exchange, routing_key='nlp-techniques'),
 )
 
 
 logger.debug("Loading spacy models for facilities...")
-model1 = ner.load_model(app.conf['NER_FACILITY_MODEL_ACK'])
-model2 = ner.load_model(app.conf['NER_FACILITY_MODEL_FT'])
-
+model1 = spacy.load(app.conf['NER_FACILITY_MODEL_ACK'])
+model2 = spacy.load(app.conf['NER_FACILITY_MODEL_FT'])
 
+logger.debug("Loading spacy model for implementing nlp techniques...")
+en_model = spacy.load(app.conf['NLP_MODEL'])
```

Comment:
The instructions given for this project were to "Use spaCy (preferred) and/or nltk", and NLTK shows lower performance for the requested ents, pos, etc so I went with spaCy. I'll add a link showing that when I get a chance. 

I had passively looked at one other option (Spark-NLP), but from the instructions and our meeting, I didn't think I was supposed to be spending time looking for new frameworks. Also, stanza requires python  > 3.6, so that's not a possible option until we upgrade from python 2.

Reply:
We also have transversal instructions such as documenting the analysis that justifies the changes and decisions made in any of the tasks. This PR relies on spaCy which was the prefered one at the moment of thinking about this project, but you have shown in the NER project that its tokenizer is worse than others and that makes me question if this is a good choice. I am looking for the analysis that you have made to justify this choice (i.e., this package and the algorithms it uses).

Regarding Python 3.6 requirements, we can call external Python 3 code from Python 2 workers as a temporary workaround until we have migrated all the code to Python 3. That is not a show stopper if we see that for instance stanza is superior.